hustle_public_api_client/samples/upload_leads_csv_with_van_integration.py

```diff
@@ -0,0 +1,145 @@
+"""
+    Usage:
+    $ python3 upload_leads_csv_with_van_integration.py BASE_URL EMAIL_ADDRESS CLIENT_SECRET ORGANIZATION_ID CSV_FILENAME
+    eg:
+    $ python3 upload_leads_csv_with_van_integration.py https://api.hustle.com/v3 jane.doe@email.com supersecretpassword 123456 ./leads_to_upload.csv
+
+    NOTE: This script expects your csv to be in the form:
+    phoneNumber,firstName,lastName,vanId
+    +12248033037,Merlin,Modlin,12345
+    +12248033090,Angella,Stearn,12346
+    +12248032747,Saran,Galiano,12347
+"""
+
+import sys
+import csv
+import time
+from hustle_public_api_client import Client
+from hustle_public_api_client import AuthenticatedClient
+from hustle_public_api_client.api.access_token import post_oauth_token
+from hustle_public_api_client.api.integrations import get_integrations
+from hustle_public_api_client.models import PostOauthTokenBody, Organization, Lead, PostLeadsBody, PatchLeadsBody, \
+    AttachLeadToIntegrationOperation, VanExternalLeadSchema, Integration
+from hustle_public_api_client.api.organizations import get_organizations
+from hustle_public_api_client.api.leads import post_leads, patch_leads
+
+if len(sys.argv) != 6:
+    print(__doc__)
+    sys.exit()
+
+base_url = sys.argv[1]
+client_id = sys.argv[2]
+client_secret = sys.argv[3]
+organization_id = sys.argv[4]
+csv_file_name = sys.argv[5]
+
+
+class Singleton(type):
+    _instances = {}
+
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
+        return cls._instances[cls]
```

Comment:
I considered it, but I felt like the added redirection wouldn't be useful to someone trying to figure out how to do the specific thing in the file. Like if someone wants to learn how to "upload leads in a csv with a van integration", I'd rather they have all of the details in a single file, rather than having to follow a redirection to a class file.